recipes/gpmdp.mdwn

```diff
@@ -0,0 +1,21 @@
+# Google Play Music Desktop Player watcher
+
+This widget can be used to display the current track information and state of [Google Play Music Desktop Player](https://www.googleplaymusicdesktopplayer.com/).
+
+1. Download `gpmdp.lua` [here](https://awesomewm.org/recipes/gpmdp.lua)
+2. Add your customization to `gpmdp.lua` starting at line 87
+    > for a in depth example, see [here](https://github.com/gregflynn/dotsanity/blob/master/awesome/widgets/gpmdp.lua)
```

Comment:
@copycat-killer I worry that the example I linked from my personal dotfiles repo will just get out of sync if pasted here and is unnecessary for the majority of the community. I included the link on the off chance that someone new to awesome might need a little more than the base example to get started